index.html

```diff
@@ -2309,6 +2309,108 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double latency
+          </dt>
+          <dd>
+            <p>
+              The estimation in seconds of audio output latency, i.e. the delay
+              between the moment the implementation sends a buffer to the
+              system and the audible sound from the audio output (speaker or
+              connector).
+            </p>
+            <p>
+              The <a><code>latency</code></a> attribute value depends on the
+              platform and it does not change for the context's lifetime.
```

Comment:
The attribute value should change in this case indeed, I'll update the description accordingly.. Thanks for pointing it out.